streams/runners/storm/common/src/main/java/com/hortonworks/streamline/streams/storm/common/StormJaasCreator.java

```diff
@@ -0,0 +1,74 @@
+package com.hortonworks.streamline.streams.storm.common;
+
+import org.apache.commons.io.FileUtils;
+
+import javax.security.auth.login.AppConfigurationEntry;
+import javax.security.auth.login.Configuration;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class StormJaasCreator {
+    public static final String STORM_JAAS_CONFIG_TEMPLATE = "storm_jaas.config_template";
+
+    private final String stormJaasConfigTemplate;
+    private final String keyTabPath;
+    private final String streamlinePrincipal;
+
+    public StormJaasCreator() {
+        URL templateFileUrl = getClass().getClassLoader().getResource(STORM_JAAS_CONFIG_TEMPLATE);
+        if (templateFileUrl == null) {
+            throw new RuntimeException("Unable to read JAAS template file for Storm.");
+        }
+
+        try {
+            stormJaasConfigTemplate = FileUtils.readFileToString(new File(templateFileUrl.toURI()), Charset.defaultCharset());
+        } catch (IOException | URISyntaxException e) {
+            throw new RuntimeException("Unable to read JAAS template file for Storm.");
+        }
+
+        Configuration configuration = Configuration.getConfiguration();
+        AppConfigurationEntry[] streamlineConfigurations = configuration.getAppConfigurationEntry("StreamlineServer");
```

Comment:
I have SrtreamlineServer hard coded in my PR too. I thought it would not be used elsewhere. But can you add it as a constant? And I will change my code to use that constant.